src/Form/AsuBrandSettingsForm.php

```diff
@@ -91,6 +91,21 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         container with that new ID will be added to your site's HTML. This
         setting is independent of whether the ASU Universal GTM is enabled."),
     ];
+    //Search settings
+    $form['asu_brand']['search_config'] = array(
+      '#type' => 'fieldset',
+      '#title' => $this->t('Search Settings'),
+    );
+    $form['asu_brand']['search_config']['asu_brand_search_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Search URL'),
+      '#default_value' => $config->get('asu_brand.asu_brand_search_url'),
+    ];
+    $form['asu_brand']['search_config']['asu_brand_local_search_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Local Search URL'),
+      '#default_value' => $config->get('asu_brand.asu_brand_local_search_url'),
```

Comment:
Add description: `If empty, your current site's base URL will be used. Optionally, you can override with the URL of your choice to be used for scoping local search. Use the format: https://yourdomain.asu.edu`